simba_ml/prediction/time_series/pipelines/transfer_learning_pipeline.py

```diff
@@ -43,7 +44,17 @@ def __evaluate_metrics(
     y_test: npt.NDArray[np.float64],
     predictions: npt.NDArray[np.float64],
     experiment_logger: wandb.WandbLogger,
+    config: transfer_learning_pipeline.PipelineConfig,
 ) -> dict[str, np.float64]:
+    if config.data.export_path is not None:
+        for i in range(predictions.shape[0]):
+            pd.DataFrame(
+                predictions[i, :, :],
+                columns=config.data.time_series.output_features,
+            ).to_csv(
+                os.path.join(os.getcwd(), config.data.export_path, f"output_{i}.csv"),
+                index=False,
+            )
```

Comment:
See above

Reply:
Changed accordingly. Please look whether this comment is still relevant.